### DISCUSSION TITLE
Memory Leak In Onnx Session Release

### DISCUSSION THREAD

**Discussion by kishorekaruppusamy** [microsoft/onnxruntime#21701] [Performance Q&A]
someone facing memory leak after doing `session.release()` in onnx-runtime web #21673 
I just found through `memlab` 

while running `memlab run --scenario scenario_file.js` normal scenario i got nothing and memory occupied in js heap is not released. 

when i run `memlab find-leaks --trace-all-objects` it shown a 638 MB object is unreferenced.

![image](https://github.com/user-attachments/assets/17481cf2-e62a-4ec4-a9dc-7cd4cb57b4c4)

Faced Similar ??

**Comment by tianleiwu** [comment]
@guschmue
